install-dependencies.sh

```diff
@@ -1,7 +1,31 @@
 #!/usr/bin/env sh
 go get -u github.com/kardianos/govendor
 go get -u github.com/go-swagger/go-swagger/cmd/swagger
-cd $GOPATH/src/github.com/zalando-incubator/pazuzu/
+PAZUZU_DIR=$GOPATH/src/github.com/zalando-incubator/pazuzu
```

Comment:
I want to suggest few changes:

* commands from `install_dependencies.sh` script are appears in `README` file, so probably we have to duplicate them there, or add info about `install_dependencies.sh`
* Move out everything related to the swagger part of this script into a separate function
* Move these constants at the beginning of the script, so it will be easy to see and change them if necessary